client/src/components/Donation/donation-modal.tsx

```diff
@@ -144,34 +144,36 @@ function DonateModal({
     }
   };
 
-  const modalHeader = (
-    <div className=' text-center block-modal-text'>
-      <Row>
-        {!closeLabel && (
-          <Col sm={10} smOffset={1} xs={12}>
-            {recentlyClaimedBlock !== null && (
-              <b>
-                {t('donate.nicely-done', {
-                  block: t(
-                    `intro:${recentlyClaimedBlock.superBlock}.blocks.${recentlyClaimedBlock.block}.title`
-                  )
-                })}
-              </b>
-            )}
-            {showMultiTier ? (
-              <h2>{t('donate.help-us-develop')}</h2>
-            ) : (
-              <b>{t(`donate.progress-modal-cta-${ctaNumber}`)}</b>
-            )}
-          </Col>
-        )}
-      </Row>
-      <Spacer size='small' />
-    </div>
-  );
+  function ModalHeader() {
```

Comment:
> We should also move the sub-components out of `DonateModal` and let them be standalone components.

I really hoped that nobody noticed, because while refactoring it, I noticed that I can make it simpler but I wanted to wait until the flag for AB test is removed. In hindsight, I should haven't opened the PR yet, like usual 🤣.

I have opened the PR anyway, so I will move them with minimal refactoring, and wait and see how it will become.

---

I have no preference between function and arrow function, depend on the usage I guess.

for now, I will use `function`, because it is the most common one used, and we can come to decided which should be used in an issue.